Makefile

```diff
@@ -30,7 +30,7 @@ dev: python
 .PHONY: web
 $(call help,make web,run just a single web worker process)
 web: python
-	@pyenv exec tox -qe dev --run-command 'gunicorn --bind :8001 --workers 1 --reload --timeout 0 --paste conf/development.ini'
+	@pyenv exec tox -qe dev --run-command 'gunicorn --bind :8001 --workers 1 --reload --reload-extra lms/templates/* --timeout 0 --paste conf/development.ini'
```

Comment:
We probably should include this in the cookiecutter, currently it's a pain to develop admin pages.

Reply:
This worked when I first tested it using just one file, it doesn't work with wildcards:

https://github.com/benoitc/gunicorn/issues/1643
https://github.com/benoitc/gunicorn/pull/1846

Which is a bit sad